src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -58,6 +62,8 @@
     @Inject
     private AnalysisService analysisService;
 
+    private List<Integer> httpSuccessCodes = Arrays.asList(HttpStatus.SC_OK, HttpStatus.SC_ACCEPTED, HttpStatus.SC_NO_CONTENT);
```

Comment:
Please add also `201` (`SC_CREATED`)
```suggestion
    private List<Integer> httpSuccessCodes = Arrays.asList(HttpStatus.SC_OK, HttpStatus.SC_CREATED, HttpStatus.SC_ACCEPTED, HttpStatus.SC_NO_CONTENT);
```